packages/genesys-spark-components/src/components/beta/gux-avatar/example.html

```diff
@@ -614,6 +624,46 @@ <h3>Anchor Tag</h3>
     </a>
   </gux-avatar-focusable-beta>
 </div>
+<h2>Change Photo (large only)</h2>
+<gux-avatar-beta name="Simon Whelehan" change-photo>
+  <img slot="image" src="https://i.pravatar.cc/300" alt="Simon Whelehan" />
+</gux-avatar-beta>
+<gux-avatar-beta
```

Comment:
The other instances of `gux-avatar` follow this API:
```
 <gux-avatar-focusable-beta>
    <a href="#"> OR <button>
      <gux-avatar-beta
        size="xsmall"
        name="Conor Darcy"
        presence="available"
        badge
        ring
      ></gux-avatar-beta>
    </a> OR </button>
  </gux-avatar-focusable-beta>
  ```
The API for the `gux-avatar-beta` component with the property `change-photo` should not be significantly different, in my opinion. Also, we need to prevent `gux-avatar-beta` with `change-photo` attribute being slotted into `gux-avatar-focusable-beta.

I would propose that the API for the `gux-avatar` photo change overlay would look something like this:

```
 <gux-avatar-focusable-beta>
    <gux-avatar-change-photo-beta>
      <gux-avatar-beta
        size="large"
        name="Conor Darcy"
        presence="available"
        badge
        ring
      ></gux-avatar-beta>
    </gux-avatar-change-photo-beta>
  </gux-avatar-focusable-beta>
  ```
Where `gux-avatar-change-photo-beta` is a button component that has the change photo event, etc. Let me know if this makes sense or if you have other opinions. I'm not as familiar with the component as you are so I might be missing something.

Reply:
I think this is really good suggestion. I've redone the PR to apply it this way.